cmd/dp-filter-api/main.go

```diff
@@ -51,12 +60,13 @@ func main() {
 
 	client := rchttp.DefaultClient
 	datasetAPI := dataset.NewDatasetAPI(client, cfg.DatasetAPIURL, cfg.DatasetAPIAuthToken)
-
+	observationStore := observation.NewStore(neo4jDriver)
+	previewDatasets := preview.PreviewDatasetStore{Store: observationStore, Limit: 20}
```

Comment:
Make the default limit configurable?

Reply:
The limit been removed and replaced with a HTTP query